### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Lordkaito**
In this branch:
1. Set the project using Ruby on Rails.
2. Set the gems needed.
3. Create migrations for each model.
4. Create controllers for each model.
5. Create different views for each controller.
6. Manage routes to match the views that were set in the controllers.
7. Style the views following the given wireframe.
8. Allow users to do some actions, like create a new recipe, add a new food or delete any of them (if the user is the owner).
9. Write tests for models and views (integration and unit tests)
10. Fix linter errors, solve conflicts and fix minor bugs found in the process.

### CODE REVIEW

File: app/views/recipes/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <input type="checkbox" disabled>
+        <%end%>
+        <span class="slider round" checked></span>
+      </label>
+    </div>
+  </div>
+  <p>Cooking time: <%= @recipe.cooking_time %> minutes</p>
+  <p><%= @recipe.description %></p>
+  <% if can? :manage, @recipe %>
+  <a class='recipe-add-ingredient' href="#openAdd">Add food</a>
```
Comment (DeliceLydia) [inline app/views/recipes/show.html.erb]: - [x] Make this like as `add ingredient` not `add food` check this wireframe
fd

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Major highlights for your project are:
- App is working well
- Clean codes
- Descriptive commit messages


## Required Changes ♻️
_Check the comments under the review._




### Cheers and Happy Rocking!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/recipes/show.html.erb (hunk 1 above)
Comment (leolpaz) [reply app/views/recipes/show.html.erb]: Done, thanks for the review!

**Review by enaburekhan — APPROVED** [review]
Hi Lordkaito, 

Thank you for making the changes suggested by the last reviewer. 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**